analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -204,21 +205,12 @@ open class Npm(
             json["author"]?.let { authorNode ->
                 when {
                     authorNode.isObject -> authorNode["name"]?.textValue()
-                    authorNode.isTextual -> parseAuthorString(authorNode.textValue())
+                    authorNode.isTextual -> parseAuthorString(authorNode.textValue(), '<', '(')
```

Comment:
Nit: I think it'd be fine to squash all commits that start to make use of the global function.